pages/teach.md

```diff
@@ -1,32 +0,0 @@
----
-layout: page-fullwidth
-title: "Teaching with The Carpentries"
```

Comment:
I would really like to link somewhere to how to become an instructor, we are not offering this service and we need to point people in the right direction.

Reply:
On every page, there's a link under "Teach" to The Carpentries' "Become an Instructor" page. See `_data/navigation.yml`.